### ISSUE TITLE
hash placed in wrong location on file

### ISSUE THREAD

**Issue by phazei** [sindresorhus/gulp-rev#244]
I have files named:
site.subsite.js

when I use rev, instead of:
site.subsite-xxx.js
I get:
site-xxx.subsite.js

Is this a bug?  Is there a work around?

**Comment by phazei** [comment]
hmm, I think this is resolved with #169

**Comment by sindresorhus** [comment]
Yeah, I think putting the hash before the last extension would be the best default behavior.

**Comment by phazei** [comment]
It did it right in v4.  I'd just upgraded to v8 today.
